### DISCUSSION TITLE
Proper way to close server

### DISCUSSION THREAD

**Discussion by avidseeker** [11ty/eleventy#1804] [Q&A]
I suppose this has to do with npm.js, but still maybe it's related to 11ty.

 I use the command `npx @11ty/eleventy --serve` to start 11ty server and I restart it using `CTRL-C`.

 However, when I restart the server (for debugging purposes), the previous serving task stays in the backgrounds

![image](https://user-images.githubusercontent.com/67508272/120092276-5c7df080-c11a-11eb-8dc5-fe578c9d7111.png)

Question:  How do I properly stop the server? because it keeps changing the port: localhost:8080, 8081, 8082, ...

**Comment by daflh** [comment]
I usually use [kill-port](https://www.npmjs.com/package/kill-port) for this task. If you don't want to install the package, just run `npx kill-port 8080`, it should works fine.
